src/map/script.cpp

```diff
@@ -24376,6 +24392,8 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF2(round, "ceil", "i"),
 	BUILDIN_DEF2(round, "floor", "i"),
 	BUILDIN_DEF(getequiptradability, "i?"),
+	
+	BUILDIN_DEF(sendmail, "?????"),
```

Comment:
why?
id, id, str, str, str, num, num
Why ??????  then? It can be a little bit confused.

Just suggestion:
iisssii